examples/opensora_cai/opensora/models/layers/blocks.py

```diff
@@ -2,13 +2,86 @@
 import numbers
 from typing import Optional, Tuple, Type
 
+import numpy as np
+
 import mindspore as ms
 from mindspore import Parameter, Tensor, nn, ops
 from mindspore.common.initializer import initializer
 
 from mindone.models.modules.flash_attention import FLASH_IS_AVAILABLE, MSFlashAttention
 
 
+class LlamaRMSNorm(nn.Cell):
+    def __init__(self, hidden_size, eps=1e-6):
+        """
+        LlamaRMSNorm is equivalent to T5LayerNorm
+        """
+        super().__init__()
+        # weight -> gamma: match the orig repo and fix the converter instead?
+        self.gamma = Parameter(np.ones(hidden_size).astype(np.float32))
+        self.variance_epsilon = eps
+
+    def construct(self, hidden_states: Tensor):
+        hidden_states = hidden_states.astype(ms.float32)
```

Comment:
not manually cast here, maybe add this in WhiteList

Reply:
A good suggestion - will add `RMSNorm` to the whitelist.